### DISCUSSION TITLE
Training problem

### DISCUSSION THREAD

**Discussion by tiomaldy** [coqui-ai/TTS#644] [General Q&A]
![image](https://user-images.githubusercontent.com/61519589/125202780-c077f680-e23a-11eb-9e08-cdd82d92b1b1.png)
Someone got this problem or its normal ?
The training stop at this point

**Comment by erogol** [comment]
It's not normal. Something got to be NaN in your model. Try disabling mixed precision training or play with the hyper parameters.

**Reply by tiomaldy** [reply]
I going to try no disable mixed hope work this time because i waste like 4 hours for this

**Reply by tiomaldy** [reply]
Can you share the config json ?

**Reply by tiomaldy** [reply]
If i change the mixed to false ,i got a CUDA error memory
